apps/web/app/signin/page.tsx

```diff
@@ -0,0 +1,116 @@
+"use client";
+
+import { useState } from 'react'; // Import useState hook
+import { useToggle, upperFirst } from '@mantine/hooks';
+import { useForm } from '@mantine/form';
+import {
+    TextInput,
+    PasswordInput,
+    Text,
+    Paper,
+    Group,
+    PaperProps,
+    Button,
+    Divider,
+    Checkbox,
+    Anchor,
+    Stack,
+} from '@mantine/core';
+
+export default function SigninPage(props: PaperProps) {
```

Comment:
I'm a bit confused with this :(

Can you explain why are we passing paper props to this component?

React actually complains about this too

![image](https://github.com/stamford-syntax-club/course-compose/assets/92321280/442a5189-8c8f-4c81-a23d-7560c808a935)